src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MethodNameCheck.java

```diff
@@ -137,6 +137,25 @@
  * }
  * </pre>
  * <p>
+ * An example of the default configuration for all methods:
+ * </p>
+ * <pre>
+ * &lt;module name="MethodName"&gt;
+ *    &lt;property name="format" value="^[a-z](_?[a-zA-Z0-9]+)*$"/&gt;
+ * &lt;/module&gt;
+ * </pre>
```

Comment:
You had to put code example for default configuration.
The default configuration is one without any other property.
This is not the default configuration since it has `format` property being explicitly written.
This is the default config:
```
<module name="MethodName"/>
```
Take help from here: https://checkstyle.sourceforge.io/config_naming.html#MethodName
See what are the default values for each property and then make some code examples accordingly.

Reply:
Resolved